content/en/tracing/trace_pipeline/generate_metrics.md

```diff
@@ -18,30 +18,21 @@ further_reading:
 
 {{< img src="tracing/apm_lifecycle/span_based_metrics.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Span-based metrics" >}}
 
-Generate metrics from 100% of ingested spans, regardless of whether the spans are indexed by a [retention filter][1].
+Generate custom metrics from ingested spans to track trends, power dashboards, and trigger monitors—even for spans that are not retained for full trace analysis.
 
-Use custom metrics for specific fixed queries and comparisons, while creating retention filters to allow arbitrary querying and investigation of the retained trace and its flame graph.
+Span-based metrics are created from spans that have been ingested by Datadog APM, regardless of whether those spans are indexed by a [retention filter][1]. These metrics allow you to extract numeric values (such as counts, durations, or custom tags) from spans and store them as long-lived [custom metrics][3] with a 15-month retention period.
 
-**Billing Note:** Metrics created from ingested spans are billed as [Custom Metrics][2].
+**Note:** The set of spans available for metric generation depends on your [APM ingestion control settings][12]. Spans dropped due to sampling or filtering are not ingested, and therefore cannot be used to generate metrics.
 
-For example, you may want to use custom metrics to visualize anomalies, create dashboards and monitors, and see trends across any parameters that are important to your business context. All generated metrics are available for 15 months as Datadog [custom metrics][3].
+Use span-based metrics when you:
```

Comment:
This list is much easier to understand than the table that was previously in the doc!